clang/lib/DPCT/CallExprRewriter.cpp

```diff
@@ -2596,6 +2593,16 @@ class CheckArgIsConstantIntWithValue {
   }
 };
 
+class CheckArgIsDefaultCudaStream {
```

Comment:
Maybe the function checkIsArgStream provides the similar function.

Reply:
I can't found `checkIsArgStream` in this repo